storage/src/lib.rs

```diff
@@ -12,6 +12,7 @@ use std::{
 };
 
 use failure::Fail;
+pub use rocksdb;
```

Comment:
@binier 
where is this need?
I dont see any usage of rocksdb on tezedge_state or other new module

Reply:
I think @vlad9486 needed it for debugger, to use same version.

Reply:
@vlad9486 
Vlad, is it needed for debugger?